main.c

```diff
@@ -1,24 +1,24 @@
 /*
-        Thymio-II Firmware
+		Thymio-II Firmware
 
-        Copyright (C) 2011 Philippe Retornaz <philippe dot retornaz at epfl dot ch>,
-        Mobots group (http://mobots.epfl.ch), Robotics system laboratory (http://lsro.epfl.ch)
-        EPFL Ecole polytechnique federale de Lausanne (http://www.epfl.ch)
+		Copyright (C) 2011 Philippe Retornaz <philippe dot retornaz at epfl dot ch>,
+		Mobots group (http://mobots.epfl.ch), Robotics system laboratory (http://lsro.epfl.ch)
+		EPFL Ecole polytechnique federale de Lausanne (http://www.epfl.ch)
 
-        See authors.txt for more details about other contributors.
+		See authors.txt for more details about other contributors.
 
-        This program is free software: you can redistribute it and/or modify
-        it under the terms of the GNU Lesser General Public License as published
-        by the Free Software Foundation, version 3 of the License.
+		This program is free software: you can redistribute it and/or modify
+		it under the terms of the GNU Lesser General Public License as published
+		by the Free Software Foundation, version 3 of the License.
 
-        This program is distributed in the hope that it will be useful,
-        but WITHOUT ANY WARRANTY; without even the implied warranty of
-        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-        GNU Lesser General Public License for more details.
+		This program is distributed in the hope that it will be useful,
+		but WITHOUT ANY WARRANTY; without even the implied warranty of
+		MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+		GNU Lesser General Public License for more details.
 
-        You should have received a copy of the GNU Lesser General Public License
-        along with this program. If not, see <http://www.gnu.org/licenses/>.
-*/
+		You should have received a copy of the GNU Lesser General Public License
+		along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
Why is there these indentation changes? Please be careful not to change indentation without reason, it adds noise to patches!